### PR TITLE
contrib/sakura: new package (3.8.7)

### PR DESCRIPTION
**Pull request by anixedn**

### CODE REVIEW

File: contrib/sakura/template.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        "pkgconf",
+        "gettext"
+]
+makedepends = ["vte-gtk3-devel"]
+depends = ["desktop-file-utils"]
+pkgdesc = "Simple but powerful libvte based terminal emulator"
+maintainer = "avgwst <avgwst@tutanota.de>"
+license = "GPL-2.0-only"
+url = "https://launchpad.net/sakura"
+source = f"https://launchpad.net/sakura/trunk/{pkgver}/+download/sakura-{pkgver}.tar.bz2"
```
Comment (nekopsykose) [inline contrib/sakura/template.py:18]: prefer .tar.gz unless there's only bz2

**Comment by nekopsykose** [comment]
you have to squash the commits manually

**Comment by anixedn** [comment]
> you have to squash the commits manually

I'm sorry, but what does that mean?

**Comment by nekopsykose** [comment]
uhh tl;dr `git rebase -i` and `squash`/`fixup` the commits so there's only the 'new package' one. i guess using_git.md in the root mentions it under 'interactive rebase'

i don't know how to explain git basics at all but maybe https://stackoverflow.com/questions/35703556/what-does-it-mean-to-squash-commits-in-git is of some use

**Comment by flukeeey** [comment]
> > you have to squash the commits manually
> 
> I'm sorry, but what does that mean?

Just FYI, how I do it is, if you have just a single commit in your branch, you can simply amend that commit and then force-push it to your branch, e.g.

```
[make changes locally]
git add <CHANGED FILES...>
git commit --amend --no-edit
git push --force
```

Force pushing is generally discouraged if others are working on your branch though, as it rewrites the commit history/hashes. But in this situation, they probably aren't. Also maybe `--force-with-lease is better` (?).

**Comment by anixedn** [comment]
thanks!

**Comment by anixedn** [comment]
so, now I think it should be fine, if it's not, please let me know!

**Comment by nekopsykose** [comment]
you should clean up the commit description too to not have needless info in it

**Comment by nekopsykose** [comment]
(also fix the format ci, by running `black` on the template)

File: contrib/sakura/template.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+build_style = "cmake"
+hostmakedepends = ["cmake", "ninja", "perl", "pkgconf", "gettext"]
+makedepends = ["vte-gtk3-devel"]
+depends = ["desktop-file-utils"]
+pkgdesc = "Simple but powerful libvte based terminal emulator"
+maintainer = "avgwst <avgwst@tutanota.de>"
+license = "GPL-2.0-only"
+url = "https://launchpad.net/sakura"
+source = f"https://launchpad.net/sakura/trunk/{pkgver}/+download/sakura-{pkgver}.tar.gz"
+sha256 = "c0feba49d88a039ce56563f43ee59a96fbf9805c02e69775a8fdc45763b8545a"
```
Comment (nekopsykose) [inline contrib/sakura/template.py:13]: i think this works with cfi too from some quick testing

File: contrib/sakura/template.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = ["cmake", "ninja", "perl", "pkgconf", "gettext"]
+makedepends = ["vte-gtk3-devel"]
+depends = ["desktop-file-utils"]
+pkgdesc = "Simple but powerful libvte based terminal emulator"
+maintainer = "avgwst <avgwst@tutanota.de>"
+license = "GPL-2.0-only"
+url = "https://launchpad.net/sakura"
+source = f"https://launchpad.net/sakura/trunk/{pkgver}/+download/sakura-{pkgver}.tar.gz"
```
Comment (nekopsykose) [inline contrib/sakura/template.py:12]:
```suggestion
source = f"{url}/trunk/{pkgver}/+download/sakura-{pkgver}.tar.gz"
```

File: contrib/sakura/template.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+pkgname = "sakura"
+pkgver = "3.8.7"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = ["cmake", "ninja", "perl", "pkgconf", "gettext"]
+makedepends = ["vte-gtk3-devel"]
+depends = ["desktop-file-utils"]
+pkgdesc = "Simple but powerful libvte based terminal emulator"
```
Comment (nekopsykose) [inline contrib/sakura/template.py:8]:
```suggestion
pkgdesc = "Libvte based terminal emulator"
```

File: contrib/sakura/template.py (hunk 2 above)
Comment (anixedn) [reply contrib/sakura/template.py:13]: yeah it works with cfi

**Pull request merged**